### PR TITLE
add possibility to add null content to entity streamed state

### PR DESCRIPTION
**Pull request by IlyaMax**

### CODE REVIEW

File: packages/relation/test/action/states/entity_streamed_state_test.dart [hunk 1]
```diff
@@ -16,12 +16,28 @@ import 'package:flutter_test/flutter_test.dart';
...
-    await entityStreamedState.content('test');
-    expect(result.map((state) => state.data).toList(), equals([null, 'test']));
+  group('EntityStreamedState content tests', () {
+    test('common case', () async {
+      final entityStreamedState = EntityStreamedState<String>();
+      final result = <EntityState<String>>[];
+      entityStreamedState.stream.listen(result.add);
+      await entityStreamedState.content('test');
+      expect(
+        result.map((state) => state.data).toList(),
```
Comment (dkrutskikh) [inline packages/relation/test/action/states/entity_streamed_state_test.dart:26]: @IlyaMax `toList` is redundant in this case

File: packages/relation/test/action/states/entity_streamed_state_test.dart [hunk 2]
```diff
@@ -16,12 +16,28 @@ import 'package:flutter_test/flutter_test.dart';
...
+      );
+    });
+    test('can put null', () async {
+      final entityStreamedState =
+          EntityStreamedState<String>(const EntityState.content('test'));
+      final result = <EntityState<String>>[];
+      entityStreamedState.stream.listen(result.add);
+      await entityStreamedState.content(null);
+      expect(
+        result.map((state) => state.data).toList(),
```
Comment (dkrutskikh) [inline packages/relation/test/action/states/entity_streamed_state_test.dart:37]: @IlyaMax and this one too

**Review by dkrutskikh — CHANGES_REQUESTED** [review]

File: packages/relation/test/action/states/entity_streamed_state_test.dart (hunk 1 above)
Comment (IlyaMax) [reply packages/relation/test/action/states/entity_streamed_state_test.dart]: +

File: packages/relation/test/action/states/entity_streamed_state_test.dart (hunk 2 above)
Comment (IlyaMax) [reply packages/relation/test/action/states/entity_streamed_state_test.dart]: +

**Review by dkrutskikh — APPROVED** [review]

**Review by trushchinskii-surf — APPROVED** [review]

**Comment by IlyaMax** [comment]
decided not to add this because we need from EntityStateBuilder to have contentBuilder with not nullable data. And no other builders should be called if isLoading and isError are false.

**Pull request closed without merging**